application/src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -222,6 +225,38 @@ public TripPattern getNewTripPatternForModifiedTrip(FeedScopedId tripId, LocalDa
     return realTimeNewTripPatternsForModifiedTrips.get(tripIdAndServiceDate);
   }
 
+  /**
+   * Get trips which have been canceled.
+   *
+   * @param feeds if not null, only return trips from these feeds
+   */
+  public ArrayList<TripOnServiceDate> getCanceledTrips(List<String> feeds) {
```

Comment:
Presumably this is an array list on purpose. Can you state it in a comment?

Reply:
I think it's because it needs to be sorted. I'm not sure how we normally handle these situations.